config/imsim.yaml

```diff
@@ -115,8 +119,8 @@ image:
                 # the main process for all of the (many) vert faint sources, which only shoot
                 # a small numer of photons.
 
-    xsize: 4096
-    ysize: 4096
+    xsize: 4072   # These are the numbers for ITL sensors, but they should be
+    ysize: 4000   # found from the camera geometry individually for each CCD.
```

Comment:
We can definitely get these from the camera_geometry. 
I think a good fairly general solution is probably to add something like the `OpsimMeta` function that can grab an arbitrary attribute from the current detector's geometry.  Something like this I think (untested):
```
def CameraValue(config, base, value_type):
    """Return some information from the camera geometry for the current detector
    """
    camera = galsim.config.GetInputObj('camera_geometry', config, base, 'CameraValue')
    det_name = base['det_name'].replace('-', '_')
    ccd = camera[self.det_name]

    req = { 'field' : str }
    opt = { 'num' : int }  # num, if present, was used by GetInputObj
    kwargs, safe = galsim.config.GetAllParams(config, base, req=req, opt=opt)
    field = kwargs['field']

    val = value_type(getattr(ccd, field))
    return val, safe

RegisterValueType('CameraValue', CameraValue, [float, int, str], input_type='camera_geometry')
```
Then in the config, it would be
```
xsize: { type: CameraValue, field: xsize }
ysize: { type: CameraValue, field: ysize }
```
And just add these attributes to the CCD class.  Or make them properties to derive from the bounds attribute, which already has this information (but in a slightly less accessible form).